chart/templates/controller.yaml

```diff
@@ -52,52 +52,59 @@ rules:
 apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRoleBinding
 metadata:
-  name: {{.Values.wds_cp_name}}-transport-controller-rolebinding
+  name: {{.Values.wds_cp_name}}-transport-controller
 roleRef:
   apiGroup: rbac.authorization.k8s.io
   kind: ClusterRole
-  name: {{.Values.wds_cp_name}}-transport-controller-role
+  name: {{.Values.wds_cp_name}}-transport-controller
 subjects:
   - kind: ServiceAccount
-    name: transport-controller-sa
-    namespace: '{{.Values.wds_cp_name}}-system'
+    name: transport-controller
+    namespace: {{.Release.Namespace}}
 ---
 apiVersion: v1
 kind: ConfigMap
 metadata:
   name: transport-controller-config
-  namespace: {{.Values.wds_cp_name}}-system
+  namespace: {{.Release.Namespace}}
 data:
-  get-kubeconfig.sh: |
-    #!/bin/bash
-    # this script receives a ControlPlane name and a parameter
-    # that determines whether to extract the ControlPlane's in-cluster kubeconfig
-    # or the external kubeconfig (if set to "true", the first will be retrieved).
-    # The function returns the requested kubeconfig's content in base64.
-    # it assumes the kubeconfig context is set to access the hosting cluster.
-
-    cpname="$1"
-    get_incluster_key="$2"
-
-    key=""
-    if [[ "$get_incluster_key" == "true" ]]; then
-      key=$(kubectl get controlplane $cpname -o=jsonpath='{.status.secretRef.inClusterKey}');
-    else
-      key=$(kubectl get controlplane $cpname -o=jsonpath='{.status.secretRef.key}');
+  get-config.sh: |
+    #!/bin/env bash

```

Comment:
Comments describing how this script is used, like in the lines being deleted, would be good